ams_flutter/lib/features/onboarding/presentation/pages/profile_student.dart

```diff
@@ -0,0 +1,182 @@
+import 'package:flutter/material.dart';
+import 'package:ams_flutter/core/constants/app_text_styles.dart';
+import 'package:ams_flutter/core/constants/app_images.dart';
+import 'package:ams_flutter/features/onboarding/presentation/widgets/bottom_navbar.dart';
+
+class ProfilePage1 extends StatefulWidget {
+  @override
+  State<ProfilePage1> createState() => _ProfilePage1State();
+}
+
+class _ProfilePage1State extends State<ProfilePage1> {
+  @override
+  Widget build(BuildContext context) {
+    return Scaffold(
+      body: SingleChildScrollView(
+        child: Padding(
+          padding: const EdgeInsets.all(20.0),
+          child: Column(
+            crossAxisAlignment: CrossAxisAlignment.center,
+            children: [
+              _buildHeader(context),
+              SizedBox(height: 16.0),
+              _buildProfileDetails(context),
+              SizedBox(height: 16.0),
+              _buildAttendance(context),
+            ],
+          ),
+        ),
+      ),
+      bottomNavigationBar: BottomNavBar(),
+    );
+  }
+
+  Widget _buildHeader(BuildContext context) {
+    return Row(
+      mainAxisAlignment: MainAxisAlignment.spaceBetween,
+      children: [
+        Image.asset(
+          AppImages.arrow_image,
+          width: MediaQuery.of(context).size.width * 0.06,
+          height: MediaQuery.of(context).size.width * 0.06,
+        ),
+        Text(
+          'Your Profile',
+          style: TextStyles.profilePageHeader,
+        ),
+        Image.asset(
+          AppImages.bell_image,
+          width: MediaQuery.of(context).size.width * 0.06,
+          height: MediaQuery.of(context).size.width * 0.06,
+        ),
+      ],
+    );
+  }
+
+  Widget _buildProfileDetails(BuildContext context) {
+    return Container(
+      padding: EdgeInsets.all(16.0),
+      decoration: BoxDecoration(
+        border: Border.all(color: Color(0xff9b9b9b)),
```

Comment:
Could use a variable to store the color value instead of hardcoding it. It must be in `app_colors.dart` file